scripts/metax_mappings/create_metax_references.py

```diff
@@ -69,5 +69,15 @@
         pass
 log.error(f"{n_matches} languages matched, and {no_matches} didn't")
 
+
+# parsing codes and identifiers for mapping fields of science
+fos_codes_url = "https://metax.demo.fairdata.fi/es/reference_data/field_of_science/_search?size=100"
```

Comment:
Are you getting all of them when `size=100`?

Reply:
There are currently 74 different FOS in the metax reference data list so that query `size` gets them all. Not sure if they'll ever add more